### DISCUSSION TITLE
Authenticate a user with flutter sdk

### DISCUSSION THREAD

**Discussion by natgross** [appwrite/appwrite#3611] [Q&A] [answered]
[With the Flutter sdk,] After creating some users, how do I authenticate a user session?
Thanks
nat

**Comment by stnguyen90** [comment, marked as answer]
After creating a user/account, the next step is to create a session. For example, you can create a session with email and password using this: https://appwrite.io/docs/client/account?sdk=flutter-default#accountCreateEmailSession
